### PR TITLE
Diagnostics tool 5119

### PR DESCRIPTION
**Pull request by eugene7646**

### CODE REVIEW

**Comment by dannysmyda** [comment]
@eugene7646 Could you also address some of the Codacy suggestions?

File: Experimental/src/org/sleuthkit/autopsy/experimental/configuration/Bundle_ja.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+AutoIngestSettingsPanel.bnTestMultiUser.text=\u30c6\u30b9\u30c8
```
[the remark this answers is not in the document]
Comment (eugene7646) [reply Experimental/src/org/sleuthkit/autopsy/experimental/configuration/Bundle_ja.properties]: According to Google, that is word "Test" in Japanaese.

**Comment by eugene7646** [comment]
@dannysmyda code review and Codacy feedback has been addressed. The remaining Codacy warnings are either erroneous or not something that we need to change.

File: Experimental/src/org/sleuthkit/autopsy/experimental/configuration/Bundle_ja.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (rcordovano) [reply Experimental/src/org/sleuthkit/autopsy/experimental/configuration/Bundle_ja.properties]: We should not be attempting translations. We have a team of Japanese students lined up for another round of localization.

**Comment by rcordovano** [comment]
@eugene7646 and @dannysmyda, it looks like there are some Codacy issues that can be addressed.

**Comment by eugene7646** [comment]
Richard I believe the remaining Codacy issues are either erroneous (ResultSet and javadocs) or not something we need to address (god class, etc)

**Comment by rcordovano** [comment]
@eugene7646, the ResultSet does not need to be closed? The generic catch block is justified?

**Comment by eugene7646** [comment]
@rcordovano That is correct. The ResultSet is getting closet by the autocloseable because we use it in try-with-resources block. The generic catch block is outside of all the specific exception catch blocks, in case there is an unexpected exception from some third party tool like Solr. It is identical to the one we have in command line ingest manager.

**Pull request merged**